server/protocols/jmap-rfc-8621-integration-tests/memory-jmap-rfc-8621-integration-tests/src/test/java/org/apache/james/jmap/rfc8621/memory/MemoryMailboxChangesMethodTest.java

```diff
@@ -35,4 +36,9 @@
             .combineWith(IN_MEMORY_SERVER_AGGREGATE_MODULE)
             .overrideWith(new TestJMAPServerModule()))
         .build();
+
+    @Override
+    public MailboxChange.State.Factory getStateFactory() {
```

Comment:
It is not a getter as you create a new one each time instead of returning a field.

How about `stateFactory()` ?